src/components/chat/chatText/chatText.ts

```diff
@@ -0,0 +1,27 @@
+import Block from 'core/Block';
+
+import './chatText.scss';
+
+type IncomingProps = {
+  text?: string;
+  type?: string;
```

Comment:
тут тип можно конкретизировать
```suggestion
  type?: "type1" | "type2" | "type3"  и тд;
```

тогда если мы передадим невалидную строку, TS выдаст нам предупреждение